plugins/org.jboss.tools.livereload.ui/src/org/jboss/tools/livereload/ui/internal/command/LaunchLiveReloadServerCommandHandler.java

```diff
@@ -0,0 +1,66 @@
+/*******************************************************************************
+ * Copyright (c) 2015 Red Hat, Inc. Distributed under license by Red Hat, Inc.
+ * All rights reserved. This program is made available under the terms of the
+ * Eclipse Public License v1.0 which accompanies this distribution, and is
+ * available at http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors: Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package org.jboss.tools.livereload.ui.internal.command;
+
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+import org.eclipse.core.commands.AbstractHandler;
+import org.eclipse.core.commands.ExecutionEvent;
+import org.eclipse.core.commands.ExecutionException;
+import org.eclipse.core.runtime.CoreException;
+import org.eclipse.core.runtime.jobs.Job;
+import org.eclipse.jface.dialogs.IDialogConstants;
+import org.eclipse.wst.server.core.IServer;
+import org.jboss.tools.livereload.core.internal.server.wst.LiveReloadLaunchConfiguration;
+import org.jboss.tools.livereload.core.internal.util.WSTUtils;
+import org.jboss.tools.livereload.ui.internal.util.Logger;
+
+public class LaunchLiveReloadServerCommandHandler extends AbstractHandler {
+
+	@Override
+	public Object execute(ExecutionEvent event) throws ExecutionException {
```

Comment:
should take current selection and try open the browser for the selected server module or selected resource (in tree or current file in editor).